Packs/Veeam/pack_metadata.json

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "Veeam App for Palo Alto Networks XSOAR",
```

Comment:
Please change to:
```suggestion
    "name": "Veeam",
```
This should fix the reoccurring IF113 validation error for all the json files.

Reply:
Hi @samuelFain, there is a package name approved by our management. We cannot change it.
Also, we don't want field names like this veeamappforpaloaltonetworksxsoarveeamnumberofstoredrestorepoints.
Can we ignore this validation in .pack-ignore?

Reply:
@nikita-konoplin Fine by me.

Reply:
done